### ISSUE TITLE
Can I do it with angular-cli? 

### ISSUE THREAD

**Issue by enough1987** [valor-software/ng2-dragula#441]
Can I do it with angular-cli?

**Comment by aaronleeucla** [comment]
Yes and no. Does not appear to be angular-cli 1.0.0-beta.17's AOT compatible. Where as, ng2-bootstrap appears to be AoT compatible. Not sure whether it's because of bug in ng2-dragular or in cli beta.17.
